src/components/MyKiva/BorrowerStatusCard.vue

```diff
@@ -213,6 +150,23 @@ const title = computed(() => `${borrowerName.value} in ${borrowerCountry.value}`
 const loanUse = computed(() => loan.value?.use ?? '');
 
 const isFundraising = computed(() => loan.value?.status === FUNDRAISING);
+const isFunded = computed(() => loan.value?.status === FUNDED);
+
+const menuOptions = computed(() => {
+	const options = [{
+		id: SHARE_ID,
+		label: 'Share',
+	}];
+
+	if (!isFunded.value) {
```

Comment:
Would it be more accurate to instead remove the option if the loan isn't fundraising? I don't know off hand which statuses don't allow comments, but we know it works for fundraising.